src/components/home/elements/IssueToken.vue

```diff
@@ -0,0 +1,752 @@
+        },
+        addresses: {
+            type: Object,
+            default: function() {},
+            require: true
+        },
+        maxSupply: {
+            type: BigNumber,
+            default: function() {
+            },
+            require: true
+        },
+        currentSupply: {
+            type: BigNumber,
+            default: function() {
+            },
+            require: true
+        },
+        walletType: {
+            type: String,
+            default: 'hotWallet',
+            require: true
+        },
+        tokenId: {
+            type: String,
+            default: '',
+            require: true
+        },
+        issuer: {
+            type: String,
+            default: '',
+            require: true
+        },
+        address: {
+            type: String,
+            default: ''
+        }
+    },
+    computed: {
+        contractId() {
+            return transaction.tokenIDToContractID(this.tokenId)
+        },
+        defaultAddress() {
+            return Vue.ls.get('address')
+        },
+        defaultColdAddress() {
+            if (this.noColdAddress) return ''
+            return Object.keys(this.coldAddresses)[0]
+        },
+        userInfo() {
+            return JSON.parse(window.localStorage.getItem(this.defaultAddress))
+        },
+        secretInfo() {
+            return JSON.parse(
+                seedLib.decryptSeedPhrase(this.userInfo.info, Vue.ls.get('pwd')))
+        },
+        seedPhrase() {
+            return seedLib.decryptSeedPhrase(this.secretInfo.encrSeed, Vue.ls.get('pwd'))
+        },
+        wordList() {
+            return this.seedPhrase.split(' ')
+        },
+        isSubmitDisabled() {
+            return !(BigNumber(this.amount).isGreaterThan(0) && this.isValidIssuer(this.address) && (this.isValidAttachment || !this.attachment) && this.isAmountValid('hot'))
+        },
+        isColdSubmitDisabled() {
+            return !(BigNumber(this.amount).isGreaterThan(0) && this.isValidIssuer(this.address) && (this.isValidColdAttachment || !this.coldAttachment) && this.isAmountValid('cold'))
+        },
+        noColdAddress() {
+            return Object.keys(this.coldAddresses).length === 0 && this.coldAddresses.constructor === Object
+        },
+        dataObject() {
+            return {
+                protocol: PROTOCOL,
+                api: this.coldApi(),
+                opc: OPC_FUNCTION,
+                address: this.address,
+                senderPublicKey: this.coldAddresses[this.address].publicKey,
+                fee: this.fee * VSYS_PRECISION,
+                feeScale: FEE_SCALE,
+                timestamp: Date.now(),
+                attachment: '',
+                contractId: this.contractId,
+                functionId: ISSUE_FUNCIDX,
+                function: transaction.prepareIssueAndBurn(BigNumber(this.amount).multipliedBy(this.tokenUnity)),
```

Comment:
functionData?

Reply:
According to https://github.com/virtualeconomy/rfcs/blob/master/text/0005-wallet-interaction-specification-3.md this field is function, not functionData.